ULMFIT.ipynb

```diff
@@ -0,0 +1,976 @@
+{
```

Comment:
<p>Colab specific paths</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Convert all Drive paths to public paths via <code>gdown</code>  please?</p>